### ISSUE TITLE
Organizers should have Dashboard

### ISSUE THREAD

**Issue by gfting** [VandyHacks/vaken#626]
Currently, Organizers do not have their Dashboard rendered. This should be fixed in our routes.

**Comment by semenodp** [comment]
@gfting can you elaborate a bit on this, including more technical specs?

**Comment by gfting** [comment]
Yeah–so in `src/client/assets/routes.js`, the OrganizerDash is supposedly rendered in for Organizers, but it currently only renders a loading spinner since the dashboard itself is broken. Therefore, `src/client/routes/dashboard/OrganizerDash.tsx` should be fixed.

**Comment by leonm1** [comment]
Implementation requirements/suggestions:

* Backend:
  * Statistics object with fields for
    * number of accepted, rejected, applied, etc hackers
    * top N chools with most hackers applied/rejected/accepted/etc.
    * class standing/grad year of applicants/attendees
    * States hackers are from
    * T shirt size aggregates
    * Number of new applicants/accepteds/rejections per day
    * Note: probably best as a stats object with fields for each status: (accepted, attending, applied, etc or other) and each field having a stats object containing the above fields
 * Frontend:
   * Display this information in graphs and pretty data viz in OrganizerDash.tsx
